### ISSUE TITLE
Braces of multi-line lambda blocks should be placed below arrow

### ISSUE THREAD

**Issue by RalfKoban** [RalfKoban/MiKo-Analyzers#526]
We should report a violation (and provide a code fix) in case the braces of multi-line lambda blocks are not placed right below the arrow.

Following example should trigger a violation:
```c#
DoSomething((_) =>
{
   // something to do here
});
```

while following example *NOT* should trigger a violation:
```c#
DoSomething((_) =>
                  {
                     // something to do here
                  });
```
